liferay-echo/liferay-portal
# LPS-143214 The ID info is not generated in the .xls file

[Link to the bug](https://issues.liferay.com/browse/LPS-143214)

Motivation
=======
Add the id of the content dashboard item to the xls generated file.

Proposed Solution
========
Create a new method in the ContentDashboardItem interface and implement it in all classes that implementing the interface.

Steps to Verify:
----------------

1. Go to Content Dashboard
2. Click on Export XLS button
3. Open the dowloaded file and see that the ID is there

Notes:
----------------
Update of the integration tests are provided.
